src/tools/build_helper/src/git.rs

```diff
@@ -94,20 +103,13 @@ pub fn get_git_modified_files(
     git_dir: Option<&Path>,
     extensions: &Vec<&str>,
 ) -> Result<Option<Vec<String>>, String> {
-    let Ok(updated_master) = updated_master_branch(git_dir) else {
-        return Ok(None);
-    };
-
-    let git = || {
-        let mut git = Command::new("git");
-        if let Some(git_dir) = git_dir {
-            git.current_dir(git_dir);
-        }
-        git
-    };
+    let merge_base = get_git_merge_base(git_dir)?;
 
-    let merge_base = output_result(git().arg("merge-base").arg(&updated_master).arg("HEAD"))?;
-    let files = output_result(git().arg("diff-index").arg("--name-only").arg(merge_base.trim()))?
+    let mut git = Command::new("git");
+    if let Some(git_dir) = git_dir {
+        git.current_dir(git_dir);
+    }
+    let files = output_result(git.arg("diff-index").arg("--name-only").arg(merge_base.trim()))?
```

Comment:
```suggestion
    let files = output_result(git.args(["diff-index", "--name-only", merge_base.trim()]))?
```
Just personal preference, not blocking in any way

Reply:
:shrug: this is existing code, I don't really care. However I see a lot of `.arg().arg()` chains throughout bootstrap so it doesn't seem like there is a consistent style either way.